### DISCUSSION TITLE
Ability to backup only once and stop the container

### DISCUSSION THREAD

**Discussion by SatishReddyBethi** [prodrigestivill/docker-postgres-backup-local#136] [Q&A] [answered]
### Feature:
Add the ability to run a postgres backup only once and stop. This feature will let us use this image in Kubernetes `initContainers` or add it as a docker dependency that only runs once before other containers can use that backup.

### Implementation:
- Include an env variable such as `BACK_UP_ONCE`. Which if set to `true` will run a backup immediately and then stop container after the backup.
- This env variable would probably be in conflict with `SCHEDULE` variable. So, my though would be to either set a priority of one over the other or do not allow these two variables to be used together.

I like this using image if not for this one missing feature. So, please, I would really appreciate it if this feature can be added.
Thank you.

**Comment by prodrigestivill** [comment, marked as answer]
You already have this option by changing the command to `/backup.sh` as is explained in the [README.md](https://github.com/prodrigestivill/docker-postgres-backup-local#manual-backups).

You can override this in k8s, docker-compose or in docker cli as in the example exposed.
